### PR TITLE
Added version to component spec

### LINKED ISSUE #644: Default components of Self-hosted mode miss the required version field
## In what area(s)?

 /area runtime

## What version of Dapr?

1.0.1

## Expected Behavior

The components auto-created by Dapr in Self-hosted mode, namely statestore.yaml and pubsub.yaml, must contain the version field that is now required for Dapr Components.

## Actual Behavior

The auto-created components do not have a version specified.

## Steps to Reproduce the Problem

Run `dapr init` and examine the components folder

## Release Note

RELEASE NOTE:

### PR DESCRIPTION
**Pull request by jigargandhi**
Added version field to pubsub.yaml and statestore.yaml

# Description

`dapr init` was not adding version to component specs generated, which is now required.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/dapr/issues/2964
closes #644 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by mukundansundar — APPROVED** [review]

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -813,6 +814,7 @@ func createRedisStateStore(redisHost string, componentsPath string) error {
 
 	redisStore.Metadata.Name = "statestore"
 	redisStore.Spec.Type = "state.redis"
+	redisStore.Spec.Version = "v1.0"
```
Comment (mukundansundar) [inline pkg/standalone/standalone.go:817]: i think this should be v1 not v1.0 ... https://docs.dapr.io/operations/support/support-versioning/#components

**Review by mukundansundar — CHANGES_REQUESTED** [review]

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (msfussell) [reply pkg/standalone/standalone.go:817]: Yes. This should be v1.

Comment (jigargandhi) [reply pkg/standalone/standalone.go]: updated to v1

**Comment by jigargandhi** [comment]
@artursouza please review this

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**